ow_plexil/rqt_plexil_plan_selection/src/rqt_plexil_plan_selection/rqt_plexil_plan_selection.py

```diff
@@ -0,0 +1,226 @@
+
+  #sets up our signal for the sub callback
+  monitor_signal = pyqtSignal(['QString']) 
+
+  def __init__(self, context):
+    '''init initializes the widget and sets up our subscriber, publisher and event handlers'''
+    super(PlexilPlanSelection, self).__init__(context)
+    self.setObjectName('PlexilPlanSelection')
+
+    # Process standalone plugin command-line arguments
+    parser = ArgumentParser()
+    parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", help="Put plugin in silent mode")
+    args, unknowns = parser.parse_known_args(context.argv())
+
+    # Find resources and Create QWidget
+    self._widget = QWidget()
+    ui_file = os.path.join(rospkg.RosPack().get_path('ow_plexil'), 'rqt_plexil_plan_selection', 'resource', 'plexil_plan_selection.ui')
+    loadUi(ui_file, self._widget)
+    self._widget.setObjectName('PlexilPlanSelectionUI')
+    if context.serial_number() > 1:
+      self._widget.setWindowTitle(self._widget.windowTitle() + (' (%d)' % context.serial_number()))
+    context.add_widget(self._widget)
+
+    #set window icon
+    icon = os.path.join(rospkg.RosPack().get_path('ow_plexil'), 'rqt_plexil_plan_selection', 'resource', 'icon.png')
+    self._widget.setWindowIcon(QIcon(icon))
+    #pub and sub setup
+    self.command_publisher = rospy.Publisher('plexil_plan_selection_commands', PlanSelectionCommand, queue_size=20)
+    self.status_subscriber = rospy.Subscriber('plexil_plan_selection_status', String, self.status_callback)
+
+    #Qt signal to modify GUI from callback
+    self.monitor_signal[str].connect(self.monitor_status)
+    #populates the plan list
+    self.populate_plan_list()
+        
+    #sets up tables
+    self._widget.sentPlansTable.setSizeAdjustPolicy(QAbstractScrollArea.AdjustToContents)
+    self._widget.planQueueTable.setSizeAdjustPolicy(QAbstractScrollArea.AdjustToContents)
+    self._widget.sentPlansTable.insertColumn(0)
+    self._widget.sentPlansTable.insertColumn(1)
+    self._widget.planQueueTable.insertColumn(0)
+    #sets up event listeners
+    self._widget.addButton.clicked[bool].connect(self.handle_add_button_clicked)
+    self._widget.removeButton.clicked[bool].connect(self.handle_remove_button_clicked)
+    self._widget.upButton.clicked[bool].connect(self.handle_up_button_clicked)
+    self._widget.downButton.clicked[bool].connect(self.handle_down_button_clicked)
+    self._widget.sendButton.clicked[bool].connect(self.handle_send_button_clicked)
+    self._widget.resetButton.clicked[bool].connect(self.handle_reset_button_clicked)
+
+  def populate_plan_list(self):
+    '''Finds all .ple plexil plans in the plans directory and stores them in the widget list.'''
+    plan_list = []
+    #get directory
+    plan_dir = os.path.join(rospkg.RosPack().get_path('ow_plexil'), 'src', 'plans')
+    #get all plans with extension .ple
+    for i in os.listdir(plan_dir):
+      if(i.endswith(".ple")):
+        plan_list.append(i.rsplit(".")[0])
```

Comment:
Replace with `p` and remove extra braces:
```suggestion
    for p in os.listdir(plan_dir):
      if p.endswith(".ple"):
        plan_list.append(p.rsplit(".")[0])
```